contracts/GrantsDAO.sol

```diff
@@ -205,11 +205,26 @@ contract GrantsDAO {
    * @param _receiver The address to receive tokens
    * @param _amount The amount to withdraw
    */
-  function withdraw(address _receiver, uint256 _amount) external onlyTeamMember () {
+  function withdraw(address _receiver, uint256 _amount) external onlyTeamMember() {
     require(_amount <= withdrawable(), "Unable to withdraw amount");
     assert(SNX.transfer(_receiver, _amount));
   }
 
+  /**
+  * @notice Allows team members to withdraw any tokens from the contract
+  * @dev Will not allow withdrawing SNX balances locked in proposals
+  * @param _receiver The address to receive tokens
+  * @param _amount The amount to withdraw
+  * @param _erc20 The address of the ERC20 token being transferred
+  *
+  */
+  function withdrawERC20(address _receiver, uint256 _amount, address _erc20) external onlyTeamMember() {
+    if (_erc20 == address(SNX)) {
```

Comment:
Just note, this checks `_erc20` isn't the same as the SNX address (proxy) . But could be circumvented by giving it the underlying Synthetix contract address.